.ci/spellcheck/.pyspelling.wordlist_anna_karenina.txt

```diff
@@ -0,0 +1,108 @@
+Alabin
```

Comment:
do we really need it?
I believe only markdown text in notebook checked by spellcheck, output is skipped

Reply:
Maybe changes from https://github.com/openvinotoolkit/openvino_notebooks/pull/1190 can help to avoid adding this word list?

I'm wondering, what happens if we will want to replace book with another

Reply:
Yes, but this is the text from README.md.

Reply:
yes, tag from Yaroslav PR can help to ignore it for checking

Reply:
Tried to skip it with the <spell> tag, but it didn't work.

Reply:
Delete added wordlist, and replace the HTML table in README.md with a screenshot.